packages/profiler/src/measure.rs

```diff
@@ -41,7 +41,10 @@ impl Measurements {
 
     pub fn compile_csv(&self, block_store: Arc<Mutex<BlockStore>>, sink: impl std::io::Write) {
         let block_store = block_store.lock().unwrap();
-        let mut wtr = csv::Writer::from_writer(sink);
+        let mut wtr = csv::WriterBuilder::new()
+            .terminator(csv::Terminator::CRLF)
+            .flexible(true)
```

Comment:
Nice. Now we can `.unwrap()` the `write_record` call

Reply:
Good call!